recipes/region/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "region" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6beb23e10eb00efd47a84d2cb9c3823512675cf29dfffb476fe7d18ab3df58cc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+
+    - pip
+  run:
+    - python>=3
```

It is necessary an empty space after ``python``
```suggestion
    - python >=3
```